src/components/elements/CommonMenuButton.tsx

```diff
@@ -162,36 +174,49 @@ const CommonMenuButton = ({
               </Stack>
             );
 
+            let menuItem;
             if (to) {
-              return (
+              menuItem = (
                 <MenuItem
                   key={key}
                   {...props}
                   component={RouterLink}
                   to={to}
                   state={linkState}
                   openInNew={openInNew}
+                  Icon={false} // omit openInNew Icon. Use menu icon instead.
+                >
+                  {menuItemLabel}
+                </MenuItem>
+              );
+            } else {
+              menuItem = (
+                <MenuItem
+                  key={key}
+                  {...props}
+                  onClick={() => {
+                    if (onClick) {
+                      // close menu before triggering onClick
+                      setAnchorEl(null);
+                      onClick();
+                    }
+                  }}
                 >
                   {menuItemLabel}
                 </MenuItem>
               );
             }
 
-            return (
-              <MenuItem
-                key={key}
-                {...props}
-                onClick={() => {
-                  if (onClick) {
-                    // close menu before triggering onClick
-                    setAnchorEl(null);
-                    onClick();
-                  }
-                }}
-              >
-                {menuItemLabel}
-              </MenuItem>
-            );
+            if (sectionLabelEl) {
```

Comment:
cant figure out how to add labels and keep accessibility of menu

Reply:
not implementing section labels from mock too much of a time sink. we probably need to implement our own menu